workloads/imagenet/imagenet_jax/workload.py

```diff
@@ -41,28 +41,6 @@ def __init__(self):
     self.dataset = 'imagenette'
     self.num_classes = 10
```

Comment:
Right now the ImageNet PyTorch workload does not use any of these attributes, because the dataset and the model are hard coded -- the same holds for all other implemented workloads (besides the ImageNet Jax workload at hand of course). If we want to add the `imagenette` dataset and a different model, I think it makes sense to either implement it as a separate `developement` workload (as you suggested [here](https://github.com/mlcommons/algorithmic-efficiency/pull/45#discussion_r789743829)) or to think of a clean way to integrate it in the ImageNet workload (which would depend on how we want to implement the "debug mode"). Either of these options should probably be implemented in a separate PR.

Reply:
Yes, I agree. We don't need these attributes for now.